### PR TITLE
Added product update for June 17th

### PR DESCRIPTION
**Pull request by sujaya-sys**

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+- **New:** Deleting a [resource definition](https://docs.humanitec.com/reference/concepts/resources/definitions) that has active resources depending on it will mark it as `Pending deletion` in the UI until all the resources associated with it are re-deployed.  
+![pending deletion](_assets/images/2022-06-17_Pending_Deletion.png)
+
+- **Changed:** Implemented a dedicated view-, add-, and edit-mode for several components to give users more control when making changes in the UI. See the [App Members](https://docs.humanitec.com/guides/manage-users-permissions/role-based-access-control#application-roles) section as an example:  
+![app members](_assets/images/2022-06-17_App_Members.png)
+
+- **Changed:** The colour contrast of the [dark- and light theme](https://docs.humanitec.com/reference/user-interface/top-bar#user-menu) according to the Web Content Accessibility Guidelines (WCAG) 2.1.
+
+- **Fixed:** Error states are no longer shared between the configuration overlays in the Accounts section of the [Organization Settings Page](https://docs.humanitec.com/reference/user-interface/organization-settings).
```
Comment (chrishumanitec) [inline docs/change-log.md:15]: I'm not sure it is actually clear what this means. Maybe rephrase to "Error messages no longer persist between the configuration..."

**Review by chrishumanitec — CHANGES_REQUESTED** [review]

File: docs/change-log.md (hunk 1 above)
Comment (sujaya-sys) [reply docs/change-log.md]: Makes sense!

**Review by christophcrichter — APPROVED** [review]
LGTM
